pkg/scheduler/actions/preempt/preempt.go

```diff
@@ -254,19 +259,20 @@ func preempt(
 			klog.V(3).Infof("No validated victims on Node <%s>: %v", node.Name, err)
 			continue
 		}
-
-		victimsQueue := util.NewPriorityQueue(func(l, r interface{}) bool {
+		// lower priority task are evicted first
```

Comment:
Those code have been refacted in another [PR sort victims in reclaim action #3389](https://github.com/volcano-sh/volcano/pull/3389/files#diff-3ef0cda85fbd1a9e76f3e873cad14fbe0cbe440718d999d0ff188413912c558aL258-R258)

Reply:
That sounds good to me, I was just about to make changes to this part of the code. I'll revert this part back later.